interactive/interactive.go

```diff
@@ -35,6 +40,17 @@ func OpenInBrowser(url string) error {
 	return err
 }
 
+func WSL2() (bool, error) {
+	if runtime.GOOS != "linux" {
+		return false, nil
+	}
+	out, err := exec.Command("cat", "/proc/version").CombinedOutput()
```

Comment:
I think we can just do grep here instead of reading the entire contents and doing `bytes.Contains`?

Reply:
I am afraid `grep` might not be thoroughly available in Linux distros by default, especially the more slim ones. `/proc/version` contains very little text, so I don't think the performance is a problem. It has this "shape": `Linux version 2.6.31.12-0.2-default (geeko@buildhost) (gcc version 4.4.1 [gcc-4_4-branch revision 150839] (SUSE Linux) ) #1 SMP 2010-03-16 21:25:39 +0100`.

Reply:
I even thought about not using `cat` and directly reading `/proc/version`. 😄

Reply:
Actually now I consider simply reading the file better. 100% agnostic of whatever is installed in the OS and hides the error from callers. If the file doesn't exist it means not `WSL 2`.